pkgs/development/libraries/libsearpc/default.nix

```diff
@@ -1,15 +1,15 @@
-{stdenv, fetchurl, automake, autoconf, pkgconfig, libtool, python2Packages, glib, jansson}:
+{stdenv, fetchFromGitHub, automake, autoconf, pkgconfig, libtool, python2Packages, glib, jansson}:
 
-stdenv.mkDerivation rec
-{
-  version = "3.0.7";
-  seafileVersion = "6.1.0";
+stdenv.mkDerivation rec {
+  version = "3.0.8";
   name = "libsearpc-${version}";
 
-  src = fetchurl
-  {
-    url = "https://github.com/haiwen/libsearpc/archive/v${version}.tar.gz";
-    sha256 = "0fdrgksdwd4qxp7qvh75y39dy52h2f5wfjbqr00h3rwkbx4npvpg";
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "libsearpc";
+    # Tag is missing: https://github.com/haiwen/libsearpc/commit/12a01268825e9c7e17794c58c367e3b4db912ad9
+    rev = "12a01268825e9c7e17794c58c367e3b4db912ad9";
+    sha256 = "00ck1hl1x0pn22q3ba32dq3ckc4nrsg58irsmrnmalqbsffhcim0";
```

Comment:
It seems that with their rolling `v3.1-latest` tag upstream wants us to ignore releases and always use the latest commit.

Reply:
Although the difference between this commit and the latest is not important; feel free to keep using it.

Reply:
I just did what Arch (AUR) does: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=libsearpc